src/modules/imageresizer/tests/Properties/AppFixture.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
```

Comment:
Stylecop is enabled but the treat warnings as errors flag hasn't been set.
On this file the warning appears since the Microsoft copyright header doesn't match the original copyright header which was set for the other files.
@crutkas How should I handle the copyright header on this file? This file was created in this PR so I added the Microsoft copyright header but stylecop throws a warning that it should match the original author's copyright header.

Reply:
tell style cop to ignore that rule for that file :)

Reply:
Added an ignore for the copyright header warning for the AppFixture.cs file and changed the project settings to treat warnings as errors.